docs/src/playground/Stocks.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from '@emotion/styled';
+import { useTheme } from '@mui/material/styles';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import React from 'react';
+
+import AppBar from './components/AppBar';
+import KeyStatistics from './components/KeyStatistics';
+import SideBar from './components/side-bar/SideBar';
+
+export default function Stocks() {
+  const theme = useTheme();
+
+  const Container = styled.div`
+    display: flex;
+    height: 100vh;
+    ${useMediaQuery(theme.breakpoints.down('lg')) ? 'flex-direction: column;' : null}
+    background-color: #f9fafa;
```

Comment:
Stray hex color, we should use one from our theme right?

Reply:
As I went I saw many more things like this - hex colors, padding spacing, font family - I get that this is the initial PR to get the example up but we spent all this time making the theme, we should really be leaning on it to do the work, right? 

I know it is a bit of a tricky ask but I want to ensure we are on the right trajectory to leverage theme as much as possible in the sample app.